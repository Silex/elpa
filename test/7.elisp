;;; Relint test file 7          -*- emacs-lisp -*-

(defun my-dolist-fun (seq)
  (let ((s ""))
    (dolist (c seq)
      (setq s (concat s (char-to-string c))))
    s))

(defun test-dolist ()
  (looking-at (my-dolist-fun '(?a ?b ?^))))

(defun my-while-fun ()
  (let ((s "")
        (c ?!))
    (while (< c ?&)
      (setq s (concat s (char-to-string c)))
      (setq c (1+ c)))
    s))

(defun test-while ()
  (looking-at (my-while-fun)))
