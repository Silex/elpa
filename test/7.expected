7.elisp:10:15: In call to looking-at: Unescaped literal `^' (pos 2)
  "ab^"
   ..^
7.elisp:21:15: In call to looking-at: Unescaped literal `$' (pos 3)
  "!\"#$%"
   ....^
